frontend/src/main/scala/bloop/cli/CommonOptions.scala

```diff
@@ -23,17 +23,33 @@ case class CommonOptions(
     @Hidden err: PrintStream = System.err,
     @Hidden ngout: PrintStream = System.out,
     @Hidden ngerr: PrintStream = System.err,
-    @Hidden env: Properties = CommonOptions.currentProperties,
+    @Hidden env: CommonOptions.PrettyProperties = CommonOptions.currentEnv,
     threads: Int = ExecutionContext.nCPUs
 ) {
   def workingPath: AbsolutePath = AbsolutePath(workingDirectory)
 }
 
 object CommonOptions {
   final val default = CommonOptions()
-  final val currentProperties: Properties = {
+
+  // Our own version of properties in which we override `toString`
+  final class PrettyProperties extends Properties {
+    override def toString: String = synchronized {
+      super.keySet().toArray.map(_.toString).mkString(", ")
+    }
+  }
+
+  object PrettyProperties {
+    def from(p: Properties): PrettyProperties = {
```

Comment:
We have `PrettyProperties` so that `toString` in `commonOptions` doesn't print the property values, only the keys, to avoid littering the screen.